enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -76,24 +76,12 @@ Prerequisites:
 
 Currently, the SDK only works with Metabase version 50.
 
-You have the following options:
-
-### 1. Running on Docker
+### Running on Docker
 
 Start the Metabase container:
 
 ```bash
-docker run -d -p 3000:3000 --name metabase metabase/metabase-enterprise:v1.50.24
-```
-
-### 2. Running the Jar file
```

Comment:
Removed. Please see [this discussion for more details](https://metaboat.slack.com/archives/C063Q3F1HPF/p1730103233091369).